recipes/armadillo/all/test_package/CMakeLists.txt

```diff
@@ -1,12 +1,32 @@
 cmake_minimum_required(VERSION 3.15)
 project(PackageTest CXX)
 
-find_package(armadillo CONFIG REQUIRED)
+find_package(Armadillo CONFIG REQUIRED)
```

Comment:
Same question here: isn't the CCI convention to use the recipe name (lowercase) ?

Reply:
In this case, no. CCI convention is to match the upstream interface. As mentioned in the description, CMake publish a FindArmadillo.cmake module, which enforces the use of `find_package(Armadillo)` and CCI convention is that the conan package matches the upstream interface wherever provided.

Reply:
Thank you for the clarification!